src/test/suite/fixtures/recorded/languages/elm/changeIfStateInk.yml

```diff
@@ -0,0 +1,32 @@
+languageId: elm
```

Comment:
This looks functionally identical to the other if state test.  What is this one supposed to be testing?